src/QMCDrivers/WFOpt/QMCFixedSampleLinearOptimize.cpp

```diff
@@ -1522,7 +1538,10 @@ void QMCFixedSampleLinearOptimize::start()
     optTarget->getConfigurations(h5FileRoot);
     optTarget->setRng(vmcEngine->getRngRefs());
     NullEngineHandle handle;
+
+    // Compute wfn parameter derivatives
```

Comment:
Please move `NullEngineHandle handle;` below this comment.

Reply:
Fixed.